src/easydiffraction/analysis/analysis.py

```diff
@@ -30,19 +31,25 @@
 class Analysis:
     _calculator = CalculatorFactory.create_calculator('cryspy')
 
-    def __init__(self, project):
+    def __init__(self, project: Any) -> None:
```

Comment:
Why use `Any` here instead of `Project`?

Reply:
because Project imports Analysis and an attempt to import Project here in order to get its signature ends up with circular import error

Reply:
but fixed with importing annotations from `__future__`.